lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -337,7 +337,7 @@ def car_api_available(
                         8, "TeslaAPI", "Car API cmd vehicles " + str(req)
                     )
                     apiResponseDict = self.json.loads(req.text)
-                except:
+                except requests.exceptions.RequestException:
```

Comment:
What about other exceptions?

Reply:
This is to be compliant with PEP 8 coding style, which doesn't allow too 'broad' exception handling.

Reply:
Understood. If we enumerate the likely exceptions thrown in the try block I am all for it, but my main concern is unhandled exceptions as a result of this which might affect the stability. I think our style changes are important but should be transparent to users.

Reply:
I agree with your concerns in regards to stability. On the other hand I'm a big fan of specific exception handling as it often reveals problems in other parts of the code that otherwise would disappear in the 'fog of broad exception handling statements'. What do you think?

Reply:
I agree, we should be explicit about the exceptions wherever we can and you've done the work of validating the likely exceptions which is all I could ask for